package.json

```diff
@@ -29,14 +29,14 @@
     "chalk": "^1.1.3",
     "chipa": "^0.1.1",
     "common-tags": "^1.4.0",
-    "data.either": "^1.5.0",
-    "data.task": "^3.1.1",
-    "immutable": "^3.8.1",
-    "immutable-ext": "^1.0.8",
+    "fluture": "^7.1.3",
+    "fluture-sanctuary-types": "^1.3.0",
     "meow": "^3.7.0",
     "ora": "^1.2.0",
     "pluralize": "^4.0.0",
-    "ramda": "^0.23.0"
+    "ramda": "^0.23.0",
```

Comment:
Are we still using Ramda somewhere?

Reply:
Yes, we could remove Ramda (`R`) in favor of Sanctuary (`S`), but `R` is a bit more flexible (i.e.
 [`S.compse`](https://sanctuary.js.org/#compose) vs [`R.compose`](http://ramdajs.com/docs/#compose) ).

For a rule of thumb I would favor `S` as much as possible, but when it makes things more complicated I'd use `R`, wdyt ? Still 99% of the cases we could use `S` so maybe we just have to stick to it.